constants.js

```diff
@@ -1,9 +1,28 @@
-const pending = '-';
+const pending = '🛑';
 const passing = '✓';
-const failed = '❌';
+const failing = '❌';
+const accessDenied = '⊘';
+const timeout = '⏱';
+const notImplemented = '—';
+
+const statusMarks = {
+  pending,
+  passing,
+  failing,
+  accessDenied,
+  timeout,
+  notImplemented,
+  // this mocha's test states aren't in present tense
+  'passed': passing,
+  'failed': failing,
+};
 
 module.exports = {
+  statusMarks,
   pending,
```

Comment:
Is exporting just `statusMarks` not sufficient here since we're already including the constants in it?

Reply:
I tried to fight it, but you were right on this one: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/7/commits/fc67a84313b7537dd7d1bb6c10dbd1744a35ba78